src/main/webapp/app/exam/participate/exercises/exercise-save-button/exercise-save-button.component.html

```diff
@@ -0,0 +1,8 @@
+<button id="save-exam" type="submit" [disabled]="submission()?.isSynced" class="btn btn-primary" (click)="onSave()">
```

Comment:
The other conditions here also include the submitted flag. Did you leave it out here on purpose?
```suggestion
<button id="save-exam" type="submit" [disabled]="submission()?.submitted && submission()?.isSynced" class="btn btn-primary" (click)="onSave()">
```

Reply:
Yes, otherwise the button will not be disabled in not started exercises because we dont have a submission at start